src/main/java/net/datafaker/providers/base/Twitter.java

```diff
@@ -51,9 +56,8 @@ public Date createdTime(boolean forward, Date base, Date constrains) {
      */
     public String twitterId(int expectedLength) {
         if (expectedLength <= 6 || expectedLength >= 25) {
-            Logger logger = Logger.getLogger(Twitter.class.getName());
-            logger.setLevel(Level.WARNING);
-            logger.warning("expectedLength <= 6 may easily cause twitter id collision. And expectedLength >= 25" +
+            LOGGER.setLevel(Level.WARNING);
```

Comment:
Does the level really have to be set on each use?

Reply:
I see it below as well

Reply:
good question...
I have no idea, I've just faced this code and don't know why it ishould log like that...

Let's assume that at least `WARN` is always switched on and remove setting of the log level